presto-main/src/test/java/com/facebook/presto/execution/TestSqlQueryQueueManager.java

```diff
@@ -1,55 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.facebook.presto.execution;
-
-import io.airlift.json.ObjectMapperProvider;
-import org.testng.annotations.Test;
-import org.weakref.jmx.MBeanExporter;
-
-import java.lang.management.ManagementFactory;
-import java.util.regex.Pattern;
-
-import static org.testng.Assert.assertTrue;
-import static org.testng.Assert.fail;
-
-public class TestSqlQueryQueueManager
-{
-    @Test
-    public void testJsonParsing()
-    {
-        parse("queue_config.json");
```

Comment:
I think the parsing tests should stay here in `presto-main` as they are unit tests, and it is better to have them in the same module to detect bugs earlier.